Test_case/conn_test2.py

```
@@ -0,0 +1,120 @@
+    {"name": "LOGIN_FAILURE", "timestamp": now(),
+     "data": {"error_code": "REG_CRED_EXPIRED", "reason": "Invalid Credentials or Unauthorized",
+              "is_online": False, "metadata": DEVICE_METADATA}},
+]
+
+
+if __name__ == "__main__":
+    print(f"Uploading {len(SAMPLE_EVENTS)} test events to {TUSD_BASE_URL} ...\n")
+    for evt in SAMPLE_EVENTS:
+        try:
+            upload_event_to_tusd(evt)
+        except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)